src/deepsparse/v2/routers/router.py

```diff
@@ -158,4 +158,5 @@ def next(
 
     @staticmethod
     def validate(ops) -> bool:
+        # TODO: still needs to be implemented for the GraphRouter
```

Comment:
What's the plan then? Do you want to implement it as a separate PR?

Reply:
yes separate